### PR TITLE
many single-backticks should be double-backticks

### PR DESCRIPTION
**Pull request by DaveWitteMorris**
The docstring of `sage.combinat.posets.posets.FinitePoset` includes:

```
``elements`` -- an optional list of elements, with ``element[i]``
      corresponding to vertex ``i``. If ``elements`` is ``None``, then it is
      set to be the vertex set of the digraph. Note that if this option is set,
      then ``elements`` is considered as a specified linear extension of the poset
      and the `linear_extension` attribute is set.
```

In the last line, this PR changes the single-backticks around `linear_extension` to double-backticks. (Currently, `linear_extension` is typeset as latex math, so it is in italics and the "e" shows up as a subscript.)

It also changes about 60 other similar problems.

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [x] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

**Review by dcoudert — APPROVED** [review]
LGTM.

**Comment by fchapoton** [comment]
More similar issues can be found using

``` gg " \`[^\`\\].[A-Za-z]*_[A-Za-z]*\`" src/sage```

**Comment by dcoudert** [comment]
indeed, and it seems we have plenty that could also be fixed in this PR...

**Comment by DaveWitteMorris** [comment]
OK, I'll spend a few minutes to do some more of these.

**Comment by DaveWitteMorris** [comment]
This PR was originally a trivial edit, but now it's a patchbomb that hits about 60 files. Will we be able to merge it?

**Review by dcoudert — APPROVED** [review]
LGTM. Thank you.

File: src/sage/combinat/crystals/spins.pyx [hunk 1]
```diff
@@ -108,7 +108,7 @@ def CrystalOfSpinsPlus(ct):
     r"""
     Return the plus spin crystal of the given type D.
 
-    This is the crystal with highest weight `Lambda_n` (the
+    This is the crystal with highest weight `\lambda_n` (the
```
Comment (user202729) [inline src/sage/combinat/crystals/spins.pyx]: The original paper (Crystal Graphs for Representations of the q-Analogue of Classical Lie Algebras
M. Kashiwara, Toshiki Nakashima) uses uppercase Lambda (Λ) for fundamental weight. (Edit the occurrence above too.)

**Review by user202729 — CHANGES_REQUESTED** [review]

File: src/sage/combinat/crystals/spins.pyx (hunk 1 above)
Comment (DaveWitteMorris) [reply src/sage/combinat/crystals/spins.pyx]: Thanks for the correction. I made the fix, so is it ok to restore to positive review?

**Review by user202729 — APPROVED** [review]

**Comment by DaveWitteMorris** [comment]
Thanks!

**Pull request merged**